.travis/before_deploy.sh

```diff
@@ -7,8 +7,7 @@ set -x
 # Travis runs the `before_deploy` stage before each deployment, but
 # we only want to build them once, as we want to use the same
 # files for both Github and PyPI
-if $(ls ./dist/afdko*.zip > /dev/null 2>&1) && \
```

Comment:
(btw, the check for .zip was wrong, since we are using the default .tar.gz format for the sdist)